### ISSUE TITLE
Incremental Search: remember selected options

### ISSUE THREAD

**Issue by minman** [notepad-plus-plus/notepad-plus-plus#2750]
Remember state of "Highlight all" and "Match case" (like default Find Dialog does).

![image](https://cloud.githubusercontent.com/assets/1247254/21640720/5c51ec88-d277-11e6-810f-5af255f38e3d.png)

**Comment by MetaChuh** [comment]
please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html) and migrate or reinstall your plugins if you update from version 7.5.9 or below.
if your issue still persists, and it is still of relevance, please feel free to file a new issue as easy to understand and reproducible as possible.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)

**Comment by minman** [comment]
@MetaChuh 
really? i have to file a new issue? why did you directly close this one? 
yes, it still persists and still of relevance. IMO it would be better if you ask first and close it later (lets say if no one responds within a week or so)

**Comment by cmeriaux** [comment]
Will be fixed by PR #7754
Actually, the **Match case** option was already working in the latest version. 
Only **Highlight all** option had to be fixed
